dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.cpp

```diff
@@ -169,6 +169,7 @@ void SegmentReadTaskPool::finishSegment(const SegmentReadTaskPtr & seg)
     if (pool_finished)
     {
         q.finish();
+        LOG_INFO(log, "pool_id={} finished", pool_id);
```

Comment:
Is it necessary?

Reply:
How about recording the overall time of `readOneBlock`, and printing it in destructor?

Reply:
`total_dmfile_read_time_ns` is the sum of all segments. The duration between the first time call `readOneBlock` and the last time `readOneBlock` returned is the real `TablesSan` duration.

And we may want to know the time cost in delta.

Reply:
ok